docs/v0.7/en/contributor-guide/overview.md

```diff
@@ -9,7 +9,7 @@
 - `Datanode` is responsible for storing data to persistent storage such as local disk or object storage in the cloud such as AWS S3, Azure Blob Storage etc.
 - `Metasrv` server that coordinates the operations between the `Frontend` and `Datanode`.
 
-![Architecture](/architecture.png)
```

Comment:
Can we just remove the `architecture.png` and the `artecture-2.png`, and `use architecture-3.png` as the new `architecture.png`?

Reply:
The documents before version 0.7 still need `architecture.png` and `architecture-2.png`.